### ISSUE TITLE
build sile.sil failed

### ISSUE THREAD

**Issue by Passw** [simoncozens/sile#130]
```
lua -v
Lua 5.3.0  Copyright (C) 1994-2015 Lua.org, PUC-Rio
```

```
 sile documentation/sile.sil
This is SILE 0.9.3-unreleased
<documentation/sile.sil>
! Font 'Gentium' not available, falling back to 'DejaVu Sans'
<documentation/macros.xml>
! Font 'Gentium Book Basic' not available, falling back to 'DejaVu Sans'

! Font 'Helvetica Neue' not available, falling back to 'Latin Modern Roman Demi'
[2] [3] [4] 
! Font 'Adobe Caslon Pro' not available, falling back to 'DejaVu Sans'
[6] [7] [8] [10] [11] [12] [13] [14] [15] [16] [18] [19] [20] 
! Font 'Scheherazade' not available, falling back to 'DejaVu Sans'
[21] [22] [23] [24] [26] <packages/rules><packages/color>[27] <packages/rotate><packages/features>[28] <packages/unichar>
Error detected:
packages/unichar.lua:14: bad argument #1 to 'char' (number has no integer representation)
```

**Comment by alerque** [comment]
The last released version (v0.9.2) had quite a few issues with Lua 5.3. These have (to the best of my knowledge) all been fixed in the current HEAD release. As of recently everything has been getting tested against several Lua versions, but that was not the case before previous releases. Are you running from the packaged source or did you build a version from the current repo?

**Comment by Passw** [comment]
@alerque Thanks for your comments. After getting the latest code, everything went well. 

```
lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 15.04
Release:	15.04
Codename:	vivid
```
